Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FlexPanel/UIElementExtensions.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace FlexPanelTest
+{
+    static class UIElementExtensions
```

Comment:
This should be removed along with the Stepper class

Reply:
Again, take a look at the update that was pushed at about the same time you made these comments.